teos/cli/teos_cli.py

```diff
@@ -54,36 +106,67 @@ def main(command, args, command_line_conf):
 
     setup_data_folder(DATA_DIR)
 
+    teos_rpc_host = config.get("RPC_BIND")
+    teos_rpc_port = config.get("RPC_PORT")
+
+    rpc_client = RPCClient(teos_rpc_host, teos_rpc_port)
+
     try:
         if command == "get_all_appointments":
-            appointment_data = get_all_appointments(config.get("RPC_BIND"), config.get("RPC_PORT"))
-            if appointment_data:
-                print(json.dumps(appointment_data, indent=4, sort_keys=True))
+            result = rpc_client.get_all_appointments()
+
+        elif command == "get_tower_info":
+            result = rpc_client.get_tower_info()
+
+        elif command == "get_users":
+            result = rpc_client.get_users()
+
+        elif command == "get_user":
+            if not args:
+                sys.exit("No user_id was given")
+            if len(args) > 1:
+                sys.exit(f"Expected only one argument, not {len(args)}")
 
+            result = rpc_client.get_user(args[0])
         elif command == "help":
             if args:
                 command = args.pop(0)
 
                 if command == "get_all_appointments":
                     sys.exit(help_get_all_appointments())
 
+                elif command == "get_tower_info":
+                    sys.exit(help_get_tower_info())
+
+                elif command == "get_users":
+                    sys.exit(help_get_users())
+
+                elif command == "get_user":
+                    sys.exit(help_get_user())
+
                 else:
                     sys.exit("Unknown command. Use help to check the list of available commands")
 
             else:
                 sys.exit(show_usage())
 
-    except (FileNotFoundError, IOError, ConnectionError, ValueError) as e:
-        sys.exit(str(e))
-    except (InvalidKey, InvalidParameter, TowerResponseError, SignatureError) as e:
-        sys.exit(f"{e.msg}. Error arguments: {e.kwargs}")
+        if result:
+            print(result)
+
+    except grpc.RpcError as e:
+        if e.code() == grpc.StatusCode.UNAVAILABLE:
+            sys.exit("It was not possible to reach the Eye of Satoshi. Check your network settings.")
```

Comment:
I'd go with something more on the lines of "is the tower running", instead of check your network settings, specially since currently the cli works has no auth, so better to be used from localhost.

Reply:
Changed in 8236a9cd754d74c618aed817765ebd11a9997d6a.